### PR TITLE
[button] feat: action button

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: examples/main/shared/editor/EditorPlugins.jsx [hunk 1]
```diff
@@ -534,6 +534,7 @@ const config = {
     //     InsertPluginButtonIcon: MyCustomIcon,
     //   },
     // },
+    // onClickEvent: () => {},
```
Comment (shaulgo) [inline examples/main/shared/editor/EditorPlugins.jsx:537]: in the editor, it should be onClick: true. You can't press a button in the editor. But the settings of the button will change accordingly.

File: packages/plugin-button/web/src/components/button-viewer.jsx [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
+    const isActionButton = Boolean(onClickEvent);
+
+    if (isActionButton) {
+      return (
+        // eslint-disable-next-line jsx-a11y/click-events-have-key-events
+        <div
+          className={this.styles.button_container}
+          style={style}
+          target={target}
+          rel={rel}
```
Comment (shaulgo) [inline packages/plugin-button/web/src/components/button-viewer.jsx:20]: ?

**Comment by oded-soffrin** [comment]
This pull request is missing two things:
- A test
- A working example (you *can* use storybook to demonstrate, it can serve as documentation as well)

**Comment by shaulgo** [comment]
I'll merge this. Make a new pr with the tests and demo.

**Pull request merged**